sdk/compute/azure-mgmt-compute/tests/test_mgmt_compute_vmss.py

```diff
@@ -982,6 +986,7 @@ def test_compute_vmss_base_2(self, resource_group):
     # can not test it, see: 
     # https://docs.microsoft.com/en-us/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-maintenance-notifications
     # """)
+    @pytest.mark.skipif(os.getenv('AZURE_TEST_RUN_LIVE') not in ('true', 'yes'), reason='only run live test')
```

Comment:
```suggestion
        @pytest.mark.live_test_only
```
Does this work? It seems more convenient